apps/web/lib/components/image-overlapper.tsx

```diff
@@ -37,11 +37,12 @@ export default function ImageOverlapper({
 	radius = 20,
 	displayImageCount = 4,
 	item = null,
-	diameter = 40,
+	diameter = 34,
 	iconType = false,
 	arrowData = null,
 	hasActiveMembers = false,
 	assignTaskButtonCall = false,
+	hasInfo = null,
```

Comment:
Please check this default Value, we can't Assign Null To String,
hasInfo has a Null as Default Value (Line 45), but his Type is a String (Line 56)